website/docs/docs/dbt-versions/core-upgrade/05-upgrading-to-v1.10.md

```diff
@@ -31,6 +31,30 @@ New features and functionality available in dbt Core v1.10
 
 Large data sets can slow down dbt build times, making it harder for developers to test new code efficiently. The [`--sample` flag](/docs/build/sample-flag), available for the `run` and `build` commands, helps reduce build times and warehouse costs by running dbt in sample mode. It generates filtered refs and sources using time-based sampling, allowing developers to validate outputs without building entire models.
 
+### Parsing `catalog.yml`
+
+dbt Core can now parse the `catalog.yml` file. This is an important milestone in the journey to supporting external catalogs for Iceberg tables, as it enables write integrations. You'll be able to provide a config specifying a catalog integration for your producer model:
+
+For example: 
+
+```yml
```

Comment:
Spec is here (dbt Internal): https://www.notion.so/dbtlabs/RFC-94-Cross-Platform-Mesh-142bb38ebda780b4befdee9b7e09902b?pvs=4#1bcbb38ebda7807b900ecb26deed725b